plugins/postfix/postfix_stats

```diff
@@ -27,6 +28,8 @@ env.pflogsumm The "pflogsumm" script, can be pflogsumm.pl if it was manually
 downloaded and installed, or "pflogsumm" if it was installed by a package 
 manager (like apt-get).
 
+Use the last part of the symlink name for grepping the hostname from log.
```

Comment:
Please provide an example here - otherwise this feature could be hard to understand.